src/managers/KeybindManager.cpp

```diff
@@ -1430,8 +1430,10 @@ void CKeybindManager::renameWorkspace(std::string args) {
             int         workspace = std::stoi(args.substr(0, FIRSTSPACEPOS));
             std::string name      = args.substr(FIRSTSPACEPOS + 1);
             g_pCompositor->renameWorkspace(workspace, name);
+            EMIT_HOOK_EVENT("rename", (std::vector<void*>{workspace, name}));
```

Comment:
first, instead of duping this, it'd be better to put this inside renameWorkspace.

second, `rename` is a terrible name. Rename what? `renameWorkspace` is a better one.

Reply:
I will modify this, thanks